pkg/v1/tkg/client/cluster.go

```diff
@@ -212,6 +212,7 @@ func getContentFromInputFile(fileName string) ([]byte, error) {
 
 func (c *TkgClient) waitForClusterCreation(regionalClusterClient clusterclient.Client, options *CreateClusterOptions, isTKGSCluster bool) error {
 	log.Info("Waiting for cluster to be initialized...")
+	options.ClusterName = "cc18"
```

Comment:
I think this is used for testing. Isn't it?

Reply:
PR was not ready to review, it is ready now you can review it again.